packages/solidity-contracts/contracts/fuelchain/FuelChainState.sol

```diff
@@ -66,7 +69,11 @@ contract FuelChainState is Initializable, PausableUpgradeable, AccessControlUpgr
 
     /// @notice Constructor disables initialization for the implementation contract
     /// @custom:oz-upgrades-unsafe-allow constructor
-    constructor() {
+    constructor(uint256 timeToFinalize, uint256 blocksPerCommitInterval) {
+        TIME_TO_FINALIZE = timeToFinalize;
+        COMMIT_COOLDOWN = uint32(timeToFinalize) * 8;
```

Comment:
what's the reason for the COMMIT_COOLDOWN to be 8 times the `timeToFinalize`?

Reply:
It is arbitrary.

As it was constant before, it was meant to be 24 hours (3 hours time to finalize * 8 = 24h).

On a second thought it should be configurable too, as depending on the config you could be setting the cooldown beyond the number of commit slots. That could be an issue.

Reply:
Ok cool, that makes sense! I agree that it should also be configurable